packages/tatum/src/wallet/wallet.ts

```diff
@@ -38,30 +38,30 @@ export const generateWallet = (
   const mnem = mnemonic ? mnemonic : generateMnemonic(256)
```

Comment:
```suggestion
  mnemonic ||= generateMnemonic(256)
```

Reply:
According to master branch the `generateAlgoWallet` takes  `mnemonic` value passed to the function directly, which means that if not provided, it skips generating mnemonic. The rest of chains uses `mnem` which contains either provided `mnemonic` or generated one if no provided and because of this difference I think `mnemonic` and `mnem` should stay separated. A potential question could be if not generating menmonic for Algo is correctly in master.